repo/packages/K/koding/0/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "2.0",
+  "name": "koding",
+  "version": "0.0.1",
+  "tags": ["mesosphere", "koding"],
+  "maintainer": "support@koding.com",
+  "description": "Koding is a web based collaborative software development tool.  We bring code development off of localhost and into the cloud.",
+  "scm": "https://github.com/koding/koding.git",
+  "website": "http://www.koding.com",
+  "framework": false,
+  "postInstallNotes": "Happy Koding!"
```

Comment:
Also add `preInstallNotes` and `postUninstallNotes`. 
For `preInstallNotes`: Add this line first - "This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.". 
Refer: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/O/openldap/3/package.json

If marathon-lb is needed for this package for work then highlight that too.